manifests/manager.pp

```diff
@@ -294,7 +294,7 @@
       }else{
         fail('This ossec module has not been tested on your distribution')
       }
-    }'Debian', 'debian':{
+    }'Debian', 'debian', 'Ubuntu', 'ubuntu':{
```

Comment:
Also, this change should be on another issue and its corresponding PR for tracking purposes.

Reply:
Hi Jose!

Thank you for reviewing!

Here I created a new issue related to such change: https://github.com/wazuh/wazuh-puppet/issues/140

Kr,

Rshad